sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/models/SerializationFormat.java

```diff
@@ -10,15 +10,15 @@
 /**
  * The list of all serialization types.
  */
-public class SerializationType extends ExpandableStringEnum<SerializationType> {
-    public static final SerializationType AVRO = fromString("avro");
+public class SerializationFormat extends ExpandableStringEnum<SerializationFormat> {
```

Comment:
[APIview suggestion:](https://apiview.dev/Assemblies/Review/be51f943baa1490597548e88861bdfd5?diffOnly=True&diffRevisionId=030d2cda43be4e6daefdfee2f97f563d) please make it final